proto/sentry_protos/snuba/v1/endpoint_trace_item_stats.proto

```diff
@@ -11,9 +11,36 @@ message AttributeDistributionsRequest {
   uint32 max_attributes = 2;
 }
 
+/*
+   This is a request for a heatmap, the x-axis is every distinct value of x_attribute,
+   the y-axis is the numerical_y_attribute divided into num_y_buckets buckets. A
+   single x,y coordinate has the count of trace-items that fit into that bucket.
+
+   Example Request:
+   x_attribute = "span.op"
+   numerical_y_attribute = "eap.duration"
+   num_y_buckets = 4
+
+   Example Response:
+   span.op = "db.query"
+    - duration 0-100ms: 10
+    - duration 100-200ms: 30
+    - duration 200-300ms: 20
+    - duration 300-400ms: 0
+
+   span.op = "http.server"
+   ...
+*/
+message HeatmapRequest {
+  string x_attribute = 1;
+  string numerical_y_attribute = 2;
```

Comment:
Why do we focus on numerical only? We should support passing any attribute and depending on the type, we'd either make up buckets ourselves or the values will be discrete and buckets are already done.